### PR TITLE
close #53 by adding updated screenshots of freesound-player's appearance

### LINKED ISSUE #53: update readme to reflect new UI updates

### PR DESCRIPTION
**Pull request by hamirmahal**
All this does is show one of `freesound-player`'s sound pages and what it looks like at its [homepage](https://freesound-player.netlify.app/) after the most recent UI changes.

I made these changes by taking screenshots of the two aforementioned pages and adding them to the repository's screenshots folder, and then linking to them from the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 ## Homepage
 
-![freesound-player's homepage as of 7:26 PM PST Thursday, August 13, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/current-appearance-of-freesound-player's-homepage-2020-08-13-192528.jpg)
+![freesound-player's homepage as of 7:49 PM PST Tuesday, September 22, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/homepage-2020-09-23-at-6.05.05-PM.png)
```
Comment (amilajack) [inline README.md:7]:
```suggestion
![freesound-player's homepage as of 7:49 PM PST Tuesday, September 22, 2020](screenshots/homepage-2020-09-23-at-6.05.05-PM.png)
```

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
 
 ## After Selecting Sound
 
-![freesound-player's appearance after selecting a sound as of 7:26 PM PST Thursday, August 13, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/appearance-of-freesound-player-after-selecting-sound-2020-08-13-192709.jpg)
+![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/soundpage-2020-09-23-at-6.07.14-PM.png)
```
Comment (amilajack) [inline README.md:11]:
```suggestion
![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](screenshots/soundpage-2020-09-23-at-6.07.14-PM.png)
```

**Review by amilajack — CHANGES_REQUESTED** [review]
Nice! Some small comments regarding how we can also simplify the links here

File: README.md (hunk 1 above)
Comment (hamirmahal) [reply README.md:7]: Ah! I didn't know we could omit the `https://github.com/amilajack/freesound-player/blob/master/` portion. Very neat.

File: README.md (hunk 2 above)
Comment (hamirmahal) [reply README.md:11]: I'll make the same change here, as well. I appreciate the tip!

Comment (amilajack) [reply README.md]: Screen Shot 2020-09-24 at 5 05 59 PM
Also I'm using GH's suggestions feature! Try it out, its super cool!

Comment (amilajack) [reply README.md]: See the "commit suggestion" dropdown

Comment (hamirmahal) [reply README.md]: I did! It lets me make the commit you suggested without having to manually make the change myself. It's really neat!

**Review by amilajack — APPROVED** [review]
Nice! 🎉

**Comment by hamirmahal** [comment]
Thanks! The [main page of the repository](https://github.com/amilajack/freesound-player) shows the updated appearance, now. It looks really nice!

Screen Shot 2020-09-24 at 5 08 53 PM

**Pull request merged**